packages/ui-extensions/src/surfaces/admin/components/AdminBlock/AdminBlock.ext.doc.ts

```diff
@@ -3,7 +3,7 @@ import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
 const data: ReferenceEntityTemplateSchema = {
   name: 'AdminBlock',
   description:
-    '`s-admin-block` is similar to the AdminBlock, providing a deeper integration with the container your UI is rendered into. However, this only applies to Block Extensions which render inline on a resource page.',
+    '`s-admin-block` is similar to the AdminAction, providing a deeper integration with the container your UI is rendered into. However, this only applies to Block Extensions which render inline on a resource page.',
```

Comment:
Instead of...

`However, this only applies to Block Extensions which render inline on a resource page.`

Maybe...

`This component should only be used in Block Extensions, which render inline on a resource page.`